extension/src/utils.js

```diff
@@ -33,8 +33,14 @@ function getLogger() {
   return logger
 }
 
+function getAuthorizationRequestHeader(request) {
+  if (request.headers) return request.headers['authorization']
```

Comment:
You can use Optional chaining everywhere 😄  https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining

```suggestion
return request?.headers?.['authorization']
```

Reply:
Optionl chaining is only compatible with node v.14 or above. Shall we change to it?

Reply:
Yes, we require explicitly node v14 or higher :) https://github.com/commercetools/commercetools-adyen-integration/blob/master/extension/package.json#L40